<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>skeleton</class>
 <widget class="QMainWindow" name="skeleton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>665</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>1000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <layout class="QHBoxLayout" name="TopBtnLayout">
        <item>
         <widget class="Line" name="vLine_div2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777192</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_pointLabe">
          <item>
           <widget class="QLabel" name="label_points">
            <property name="text">
             <string>Points on picture</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_points"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="tech_calc_btnLayoy">
          <item>
           <widget class="QLabel" name="label_tech">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="text">
             <string>Technical buttons</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_reset">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reset all the selected points and calculated variables&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Zoom">
            <property name="text">
             <string>Zoom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sld_zoom">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Zoom Control&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="accessibleName">
             <string/>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_calc">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="text">
             <string>Calculate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_trackMom">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Calculate Track momentum&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Cal Track Momemetum</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_optDen">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Calculate Optical Density&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Calc Optical Density</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_angle">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Calculate Opening Angle&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Caculate Angle</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="vSpacer_tech_calcLayout">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="usrSel_layout">
          <item>
           <widget class="QLabel" name="label_userSle">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="text">
             <string>User selection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_placeMar">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Place a new marker on the track&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Place track marker</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="dlForm">
            <property name="labelAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="setDlLabel">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>set DL</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="setDlLineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btn_drwAngle">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Draw reference for angle&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Draw angle ref.</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="vSpacer_userSel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777192</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777192</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ConsoleLayout">
          <item>
           <widget class="QLabel" name="label_console">
            <property name="text">
             <string>Console</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser_consoleOutput">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="vSpace_pic_topbtnLayou">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777192</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="picLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>312</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777192</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
